packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -210,6 +212,47 @@ export default class VList {
         this.rootList = null;
     }
 
+    /**
+     * Splits the VList in two, using the element provided in the parameter as separator
+     * Write the result back into DOM tree
+     * After that, this VList becomes unavailable because we set this.rootList to null
+     * @param separator The HTML element that indicates when to split the VList
+     * @param startNumber The number that the new list is going to start
+     */
+    splitWriteBack(separator: HTMLElement, startNumber: number) {
```

Comment:
I wish the API should be something like:
```ts
const vlist = new VList(...);
vlist.setStartNumber(li, num);
vlist.writeBack();
```
So that it can match the pattern of existing VList functions.